SmartSearch/Field.cs

```diff
@@ -25,7 +26,10 @@ public Field(string name, FieldType type, FieldRelevance relevance, bool enableF
         {
             Name = name;
             Type = type;
+
             Relevance = relevance;
+            RelevanceBoost = FieldRelevanceBoost.GetBoostValue(relevance);
```

Comment:
Calculates the relevance boost when the field is created, rather than leaving it to be calculated elsewhere on demand.